xcode/Subconscious/Shared/Components/Common/Buttons/CloseButtonView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CloseButtonView.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 3/3/23.
+//
+
+import SwiftUI
+
+/// Circle-X close button used in some sheets.
+/// Sight-matched in Figma to close button from Apple Notes.
+struct CloseButtonView: View {
+    var action: () -> Void
+
+    var body: some View {
+        Button(
+            action: action,
+            label: {
+                Image(systemName: "xmark")
+                    .font(.system(size: 16, weight: .semibold))
+                    .frame(width: 30, height: 30)
+                    .foregroundColor(.secondary)
+                    .background(Color.secondaryBackground)
+                    .clipShape(Circle())
+                    .frame(width: 44, height: 44)
```

Comment:
Added `AppTheme.minTouchSize` for 44.

I'm wary of adding constants for anything below 44, since these are only used in bespoke cases, and should only be used carefully in bespoke places.